unix/qstrdefsport.h

```diff
@@ -79,6 +79,7 @@ Q(listen)
 Q(accept)
 Q(recv)
 Q(recvfrom)
+Q(settimeout)
```

Comment:
Adding a Q(setblocking) line causes memory to be allocated to hold the qstr. So, the general idea is that for features which can be turned on or off then there should be an #ifdef around the qstr definitions as well, something like: https://github.com/micropython/micropython/blob/master/stmhal/qstrdefsport.h#L215-L245